File: src/main/scala/com/bombproj/model/Log.java
package com.bombproj.model;

import com.queryflow.annotation.Table;
import lombok.Data;

import java.io.Serializable;
import java.util.Date;

@Data
@Table("log")
public class Log implements Serializable {

    private static final long serialVersionUID = 1L;

    private String id;
    private Integer state;
    private Date createTime;
    private String modelClass;
    private String modelName;
    private String type;
    private String updateUserId;
    private String updateUserName;
    private String remark;
    private String content;

}

File: src/main/scala/com/bombproj/model/Log.scala
package com.bombproj.model

import java.util.Date

import com.queryflow.annotation.Table

import scala.beans.BeanProperty

@SerialVersionUID(1L)
@Table("log")
case class Log() {

    @BeanProperty var id: String = _
    @BeanProperty var state: Int = _
    @BeanProperty var createTime: Date = _
    @BeanProperty var modelClass: String = _
    @BeanProperty var modelName: String = _
    @BeanProperty var `type`: String = _
    @BeanProperty var updateUserId: String = _
    @BeanProperty var updateUserName: String = _
    @BeanProperty var remark: String = _
    @BeanProperty var content: String = _

}
